README.md

```diff
@@ -15,7 +15,7 @@ This project contains Terraform scripts to provision the AWS cloud infrastructur
 [<img src="./docs/images/viya4-iac-aws-diag.png" alt="Architecture Diagram" width="750"/>](./docs/images/viya4-iac-aws-diag.png?raw=true)
 
 This project helps you to automate the cluster-provisioning phase of SAS Viya deployment. To learn about all phases and options of the
-SAS Viya deployment process, see [Getting Started with SAS Viya and Azure Kubernetes Service](https://go.documentation.sas.com/doc/en/itopscdc/default/itopscon/n1d7qc4nfr3s5zn103a1qy0kj4l1.htm) in _SAS&reg; Viya&reg; Operations_.
+SAS Viya deployment process, see [Getting Started with SAS Viya and Amazon Elastic Kubernetes Service (EKS)](https://go.documentation.sas.com/doc/en/itopscdc/v_025/itopscon/p111dd1k5z99kin1f7mciggn9wop.htm) in _SAS&reg; Viya&reg; Operations_.
```

Comment:
Instead of v_025, shouldn't we use default?  v_025 points to 2021.2.5 doc (v_024 points to 2021.2.4, etc.) My understanding is that "default" points to the latest.